components/ocf-host-application/AboutOurFees.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { ArrowLeft2 } from '@styled-icons/icomoon/ArrowLeft2';
+import { ArrowRight2 } from '@styled-icons/icomoon/ArrowRight2';
+import { FormattedMessage } from 'react-intl';
+
+import Container from '../Container';
+import { Box, Flex } from '../Grid';
+import Illustration from '../home/HomeIllustration';
+import Link from '../Link';
+import StyledButton from '../StyledButton';
+import { H1, P } from '../Text';
+
+import OCFPrimaryButton from './OCFPrimaryButton';
+
+const AboutOurFees = () => (
+  <Flex flexDirection="column" alignItems="center" justifyContent="center" mt={['24px', '48px']}>
+    <Flex flexDirection={['column', 'row']} alignItems="center" justifyContent="center" mb={[null, 3]}>
+      <Box width={'160px'} height={'160px'} mb="24px">
+        <Illustration
+          alt="About our fees illustration"
+          src="/static/images/ocf-host-application/aboutOurFees-illustration.png"
+        />
+      </Box>
+      <Box textAlign={['center', 'left']} width={['288px', '404px']} mb={4} ml={[null, '24px']}>
+        <H1
+          fontSize="32px"
+          lineHeight="40px"
+          letterSpacing="-0.008em"
+          color="black.900"
+          textAlign={['center', 'left']}
+          mb="14px"
+        >
+          <FormattedMessage id="OCFHostApplication.aboutOurFees" defaultMessage="About our fees" />
+        </H1>
+        <P fontSize="16px" lineHeight="24px" fontWeight="500" color="black.700">
+          <FormattedMessage
+            id="OCFHostApplication.aboutOurFees.description"
+            defaultMessage="In order to support you better, we use the Open Collective Platform to manage your initiative."
+          />
+        </P>
+      </Box>
+    </Flex>
+    <Flex justifyContent="center" alignItems="center" my={3} flexDirection={['column', 'row']}>
+      <Container
+        width="248px"
+        height="172px"
+        borderRadius="8px"
+        border="1px solid #C4C7CC"
+        display="flex"
+        flexDirection="column"
+        alignItems="center"
+        justifyContent="center"
+        padding="24px"
+        mb={['24px', 0]}
+        mr={[null, '48px']}
+      >
+        <P color="#05464A" fontSize="28px" lineHeight="36px" fontWeight="500" mb={2}>
+          5%
```

Comment:
Good point but what about the rest of the percentage fees on the page when they change? 🤔

Reply:
Can't do much for them indeed, they're not stored in the DB. Since the page has no dynamic data from GraphQL, let's not make this more complicate - hardcoding here is fine.